### PR TITLE
Added simple captcha

### PR DESCRIPTION
**Pull request by k-nut**
First draft for  #7 feedback is much appreciated!

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -48,6 +48,10 @@ class RequestForm(Form):
             ]
             )
     email_confirm = TextField('Confirm Email')
+    captcha = TextField(
+            'Capital of Germany (to make sure that you are human)',
+            validators=[validators.AnyOf('Berlin', 'berlin')]
```
Comment (booo) [inline app.py]: Maybe we need to add a Required here? I can leave this field blank at the moment.

File: templates/_formhelpers.html [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+    {% if field.errors %}
+      <ul class=errors>
+      {% for error in field.errors %}
+        <li>{{ error }}</li>
+      {% endfor %}
+      </ul>
+    {% endif %}
+  {% else %}
+    {% if field.errors %}
+      Not correct. Please try again. If you have any more problems contact the mailing list.
```
Comment (booo) [inline templates/_formhelpers.html]: I don't understand why we need another UI design for the captcha error.

Comment (k-nut) [inline templates/_formhelpers.html]: If we do not do this and the user inputs something that is not correct the form will show the allowed values as errors. So below the field 
- Berlin

will appear.

Comment (booo) [inline templates/_formhelpers.html]: Can't we use the message property in `validators.anyof()`?

http://wtforms.simplecodes.com/docs/0.6/validators.html

**Pull request merged**